src/renderer/ducks/settings/sagas.ts

```diff
@@ -14,7 +15,12 @@ function* loadAccountSettings(): Generator<PutEffect | Promise<Settings | null>,
 function* saveAccountSettings(action: {payload: ISettings}): Generator<PutEffect | Promise<void>> {
     yield database.settings.set(DEFAULT_ACCOUNT, action.payload);
 
-    if (action.payload.reporting !== undefined) yield put(setReporting(action.payload.reporting));
+    if (action.payload.reporting !== undefined) {
```

Comment:
Wouldn't this prevent changing reporting from true to false or reverse?

Reply:
`ISettings` all params are optional and this cover a case if is reporting not provided